web/sites/default/config/fixed_block_content.fixed_block_content.footer_accessibility_statement.yml

```diff
@@ -5,8 +5,8 @@ dependencies:
+default_content: '{"_links":{"self":{"href":"https:\/\/portlandor.lndo.site\/block\/13?_format=hal_json"},"type":{"href":"http:\/\/fixed_block_content.drupal.org\/rest\/type\/block_content\/basic"}},"langcode":[{"value":"en","lang":"en"}],"type":[{"target_id":"basic"}],"status":[{"value":true,"lang":"en"}],"info":[{"value":"Footer accessibility statement","lang":"en"}],"reusable":[{"value":true}],"default_langcode":[{"value":true,"lang":"en"}],"revision_translation_affected":[{"value":true,"lang":"en"}],"metatag":[{"value":{"geo_placename":"Portland","geo_position":"45.515012;-122.679096","geo_region":"US-OR","canonical_url":"https:\/\/portlandor.lndo.site\/block\/13","og_site_name":"Portland.gov","og_url":"https:\/\/portlandor.lndo.site\/block\/13","twitter_cards_type":"summary","twitter_cards_page_url":"https:\/\/portlandor.lndo.site\/block\/13"}}],"body":[{"value":"\u003Ch2 class=\u0022sr-only\u0022\u003EAccessibility statement\u003C\/h2\u003E\r\n\r\n\u003Cp\u003E\u003Cspan\u003EThe City of Portland ensures meaningful access to City programs, services, and activities to comply with Civil Rights Title VI and ADA Title II laws and reasonably provides: translation, interpretation, modifications, accommodations, alternative formats, auxiliary aids and services. To request these services, contact \u003Ca href=\u0022tel:503-823-4000\u0022\u003E503-823-4000\u003C\/a\u003E or TTY via\u0026nbsp;Oregon Relay Service:\u0026nbsp;\u003Ca href=\u0022tel:711\u0022\u003E711\u003C\/a\u003E.\u003C\/span\u003E\u003C\/p\u003E\r\n","format":"full_html","processed":"\u003Ch2 class=\u0022sr-only\u0022\u003EAccessibility statement\u003C\/h2\u003E\n\n\u003Cp\u003E\u003Cspan\u003EThe City of Portland ensures meaningful access to City programs, services, and activities to comply with Civil Rights Title VI and ADA Title II laws and reasonably provides: translation, interpretation, modifications, accommodations, alternative formats, auxiliary aids and services. To request these services, contact \u003Ca href=\u0022tel:503-823-4000\u0022\u003E503-823-4000\u003C\/a\u003E or TTY via\u00a0Oregon Relay Service:\u00a0\u003Ca href=\u0022tel:711\u0022\u003E711\u003C\/a\u003E.\u003C\/span\u003E\u003C\/p\u003E\n","summary":"","lang":"en"}],"field_title":[{"value":"Footer accessibility statement"}]}'
```

Comment:
Huh... this was incorrect before this story, but I wonder if you could take a second and clean up the absolute URLs that point to our localdev? 😳

Reply:
The way fixed block content works, you put in defaults and then trigger the defaults to take over the content after deployment. Maybe a quick demo of how that works is in order. I'll ping you in Slack in a few.